src/Microsoft.DotNet.Compiler.Common/Executable.cs

```diff
@@ -85,6 +85,7 @@ private static void CopyAllDependencies(string outputPath, IEnumerable<LibraryEx
                 libraryExport.RuntimeAssemblies.CopyTo(outputPath);
                 libraryExport.NativeLibraries.CopyTo(outputPath);
                 libraryExport.RuntimeAssets.StructuredCopyTo(outputPath);
+                libraryExport.CompilationAssets.StructuredCopyTo(outputPath);
```

Comment:
Put some comments on what you think it might be and why the structured copy is required. I wouldn't expect that for the pdbs.